app/src/main/res/values/strings.xml

```diff
@@ -445,6 +445,7 @@
     <string name="play_queue_stream_detail">Details</string>
     <string name="play_queue_audio_settings">Audio Settings</string>
     <string name="hold_to_append">Hold to enqueue</string>
+    <string name="enqueue_stream">Enqueue stream</string>
     <string name="enqueue_on_background">Enqueue in the background</string>
     <string name="enqueue_on_popup">Enqueue in a popup</string>
```

Comment:
I am not sure, but if these are unused, please remove them.

Reply:
We just need "Enqueue", no matter which button is long-pressed.

Reply:
That's true

Reply:
Probably yes, and if you do that please also rename the StreamDialogEntry item from `enqueue_stream` to just `enqueue`

Reply:
@Stypox I can rename the enum, but I can't rename the string resource since there is also `R.string,.enqueue` (which value is "Queue"), so I'd keep the string resource as `enqueue_stream` but change the value to "Enqueue". Okay?

Reply:
Mmmh, having the value "Queue" for `R.string.enqueue` does not seem right, though changing it would create problems with translations. So yeah, do as you said ;-)

Reply:
By the way, regarding strings: 
I just noticed that `NavigationHelper#enqueueOnBackgroundPlayer` and `enqueueOnPopupPlayer` show specific Toast-messages (`R.string.popup_playing_append` and `R.string.background_player_append`). They should be replaced/unified, too. For example with `R.string.player_append`. What do you think?

Reply:
Yup.